backend/project_review_coordination/models.py

```diff
@@ -56,4 +60,24 @@ def get_projects_user_can_review(user):
             .prefetch_related("content_item", "recruit_project")
         )
 
-        return cards
+        if user.is_superuser:
+            return cards
+        
+        permitted_cards = []
+        perm_checker = ObjectPermissionChecker(user)
+        
+        for card in cards:
+            assignee = card.assignees.first()
+            assignee_teams = assignee.teams()
+            
+            if len(assignee_teams) == 0:
+                continue  # prefetch_perms will fail if there are no teams
+
+            perm_checker.prefetch_perms(assignee_teams)
+            for view_permission in Team.PERMISSION_VIEW:
+                if any(
+                    (perm_checker.has_perm(view_permission, team) for team in assignee_teams)
+                ):
+                    permitted_cards.append(card)
+    
+        return permitted_cards
```

Comment:
Is [this](https://github.com/Umuzi-org/Tilde/blob/f9172d81893ae25e99eddc2f287868b1bd8d028a/backend/frontend/views.py#L604) not a slicker way of writing the above. Then we wouldn't need to explicitly looper over perms